### ISSUE TITLE
Create automation card for True GitHub Contributors 

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [100Automations/Website#31]
### Overview
Provide collateral for the OpenSourceAutomations.org website

### Action Items
- [x]   Gather items
  - [x]  600 x 400 image
     - [x]  Alt image text
  - [x]  Name of automation
  - [x]  A blurb about the automation
  - [x]  Links (github, slack channel url, Testing Site, Live Site, etc.)
  - [x] Healthy project files (see resources for instructions)
     - [x]  Readme.md URL
     - [x]  contributing.md URL
     - [x]  security.md URL
     - [x]  support.md URL
  - [x] Current status of Automation
     - [ ] in use and ready to deploy
     - [x] in use and being refactored
     - [ ] ready to deploy
     - [ ] in development
     - [ ] ready for development
     - [ ] in ideation
     - [ ] abandoned
  - [x] If status in use, what codebase can it be seen in?
   - [x] Help wanted (check all that apply)
     - [ ] needs maintenance
     - [ ] needs support for users
     - [ ] needs refactoring
     - [ ] needs documentation 
     - [x] needs peer review
     - [ ] needs to be built
     - [ ] needs to be researched
     - [ ] needs advice
     - [ ] needs takeover
        - [ ] Advice Needed: (type in sort description of what you need)
- [x]   Add automation Card to website

### Resources/Instructions
See project cards on the hackforla.org website for general idea, although the format of these will be different.

- [GitHub's guide: Setting guidelines for repository contributors](https://docs.github.com/en/articles/setting-guidelines-for-repository-contributors)
- [GitHub's guide: Adding a security policy to your repository
](https://docs.github.com/en/articles/adding-a-security-policy-to-your-repository)
- [GitHub's guide: Adding support resources to your project
](https://docs.github.com/en/articles/adding-support-resources-to-your-project)

**Readme should include:**
- Overview 
  - _What kind of repetitive thing do you have to do often and what is the benefit of automating it?_
- Current State 
  - _"As-is" most likely something manual but could be partially automated_
- Future Development
  - _The “To-be” state_
- Stakeholders
  - _Impact - who benefits and how?_
- Anticipated outcomes
- Resources/Instructions
  - _How to run the automation and any packages or dependencies_ 
- Language
  - _Programming language(s)_
- Platform
  - _Where is it deployed (e.g. GitHub, Linux Command Line, Windows Command Line etc)_
- Automation triggers 
  - _What starts it? What's it responding to?
  - _Is it Time-based (Specify frequency (e.g. 1x/week)?_
  - _Is it Event-based (e.g. someone just created a new GitHub on a repo)?_
- Input required 
  - _How much manual or custom input is required?_
- Output 
  - _What's the desired result? What do we not want to see?_

**Comment by ExperimentsInHonesty** [comment]
![Screen Shot 2020-08-24 at 2 03 06 PM](https://user-images.githubusercontent.com/37763229/91096384-cd83fb00-e612-11ea-9592-d06c17c37138.png)
Alt text : Some GitHub profile Images of people who have contributed to the hackforla.org website

**Comment by KianBadie** [comment]
Will fill in blank fields when I have them.

@ExperimentsInHonesty should I create/add the repo to 100 Automations once it is done? Or can it be a repo with active development on the unfinished project? Because right now, the repo and readme links below do not exists, but will once the repo is created. Also, since this tool isn't being used in the website yet (but the idea behind it is being used), should I still have the the "if status in use, used in codebase..." field be to hackforla?  

**600x400 Image:** 
![Screen Shot 2020-08-24 at 2 03 06 PM](https://user-images.githubusercontent.com/37763229/91096384-cd83fb00-e612-11ea-9592-d06c17c37138.png)
**Alt text:**  Some GitHub profile Images of people who have contributed to the hackforla.org website

**Name of automation:** True GitHub Contributors
**Blurb:** Node module used for fetching new data about GitHub project contributors using octokit/rest.js. Can fetch contributor data sets based on issue comments, across an organization, and since certain dates.
**Links:**
- Repo: github.com/100automations/true-github-contributors

**Resources:**
- Readme: https://github.com/100automations/true-github-contributors/blob/main/README.md

**Looking for skills wise:** Nodejs, JavaScript, Code Review
 **Status:** In development
 **If status in use, used in codebase**: HackforLA.org https://github.com/hackforla/website

**Comment by ExperimentsInHonesty** [comment]
Q1: should I create/add the repo to 100 Automations once it is done? Or can it be a repo with active development on the unfinished project?
A1: Yes, create it now

Q2: since this tool isn't being used in the website yet (but the idea behind it is being used), should I still have the the "if status in use, used in codebase..." field be to hackforla?
A2: Yes and I refactored the way we ask about it.  So I have checked off the options above
 Status of Automation:  in use and being refactored 


Bonnie Q1: Does the wording " and since certain dates." seem right.  Is there a more technical way to express that? Like time range or from X date forward, etc?
Bonnie Q2: You put the following" Looking for skills wise: Nodejs, JavaScript, Code Review". I have added some extra check boxes to the top section: https://github.com/100Automations/Website/issues/31#issue-684965182. And I checked off Peer Review.  We will also use the automated ability to pull language data from repository that we use the hack for LA website.  So given that.  Do you need other help besides code review?  For instance, help with Node.js or Javascript specifically?

**Comment by KianBadie** [comment]
@ExperimentsInHonesty I have created my repository for 100 Automations. It has a read.me but it is very outdated.

**Comment by ExperimentsInHonesty** [comment]
Will edit fields as necessary.

@ExperimentsInHonesty should I create/add the repo to 100 Automations once it is done? Or can it be a repo with active development on the unfinished project? Because right now, the repo and readme links below do not exists, but will once the repo is created. Also, since this tool isn't being used in the website yet (but the idea behind it is being used), should I still have the the "if status in use, used in codebase..." field be to hackforla?

600x400 Image:
Screen Shot 2020-08-24 at 2 03 06 PM
Alt text: Some GitHub profile Images of people who have contributed to the hackforla.org website

Name of automation: True GitHub Contributors
Blurb: Node module used to extend GitHub's API Client octokit/rest.js to fetch new types of contributor data sets. Can fetch contributor data sets based on issue comments, across an organization, and since certain dates.
Links:

Repo: github.com/100automations/true-github-contributors
Resources:

Readme: https://github.com/100Automations/true-github-contributors/blob/mixin/readme.md
Looking for skills wise: Nodejs, JavaScript, Code Review
Status: In use and being refactored
If status in use, used in codebase: HackforLA.org https://github.com/hackforla/website

**Comment by KianBadie** [comment]
I updated the [readme](https://github.com/100Automations/true-github-contributors/blob/mixin/readme.md) and created first drafts for the [support](https://github.com/100Automations/true-github-contributors/blob/mixin/SUPPORT.md), [security policy](https://github.com/100Automations/true-github-contributors/blob/mixin/SECURITY.md), and [contributing](https://github.com/100Automations/true-github-contributors/blob/mixin/CONTRIBUTING.md) docs.

There used to be technical documentation for the readme, but I removed it because the project has changed so much since then that the docs were no longer correct and did not provide useful instructions. I will add detailed docs once testing is complete (hopefully by this week).

**Comment by ExperimentsInHonesty** [comment]
@CamilaArias @NivenPrasad Here is the finished project card for True GitHub Contributors.  Let's discuss at next meeting.

**Comment by ExperimentsInHonesty** [comment]
**600x400 Image:** 
![Screen Shot 2020-08-24 at 2 03 06 PM](https://user-images.githubusercontent.com/37763229/91096384-cd83fb00-e612-11ea-9592-d06c17c37138.png)
**Alt text:**  Some GitHub profile Images of people who have contributed to the hackforla.org website

**Name of automation:** True GitHub Contributors
**Blurb:** Node module used to extend GitHub's API Client octokit/rest.js to fetch new types of contributor data sets. Can fetch contributor data sets based on issue comments, across an organization, and from X date forward.
**Links:**
- Repo: github.com/100automations/true-github-contributors

**Resources:**
- Readme: https://github.com/100Automations/true-github-contributors/blob/mixin/readme.md
 - contributing.md https://github.com/100Automations/true-github-contributors/blob/mixin/CONTRIBUTING.md
 - security.md https://github.com/100Automations/true-github-contributors/blob/mixin/SECURITY.md
 - support.md  https://github.com/100Automations/true-github-contributors/blob/mixin/SUPPORT.md

**Help wanted:** Peer Review
 **Status:** In use and being refactored
 **Used in codebase**: HackforLA.org https://github.com/hackforla/website

**Comment by CamilaArias** [comment]
Should we add **languages** here since we are now showing them?
**Healthy files** aren't being shown on the cards.
**Links**, we're only showing the link to the repo through the Readme button.
For the **codebase** should we specify we need the link to the repo and logo?

**Comment by ExperimentsInHonesty** [comment]
Here is the final details for this project

![true-github-contributors-3](https://user-images.githubusercontent.com/37763229/92680204-258f5280-f2df-11ea-9946-27d222bddf15.png)

**Alt text:**  Some GitHub profile Images of people who have contributed to the hackforla.org website and titles under some names indicating Designer, Stakeholder, Product all in uppercase text.

**Description:** GitHub’s Contributor’s endpoint only shows the people who put in pull requests.  On cross functional teams that use GitHub to manage projects, contributions from non code contributors (Designers, Product/Project Managers, Business Analysts, Q/A testers, etc.) are represented by comments on issues.   

True GithHub Contributors delivers a consolidated list of who contributed and how many times.  This code can be seen on Hack for LA’s project pages, where are contributors are represented by their GitHub Avatar in the Contributors section.

**Status:** IN USE AND BEING REFACTORED

**As seen on:** HackforLA (which should link to https://github.com/hackforla/website

**Help Wanted:** Needs peer review

**Languages:** Javascript

**Comment by NivenPrasad** [comment]
Hi @CamilaArias - please make an extra card on the Figma board that reflects the information in [the comment directly above](https://github.com/100Automations/Website/issues/31#issuecomment-706579504).

**Comment by Olivia-Chiong** [comment]
Progress: None
Blockers: ?
Availability: ?
ETA: ?

@amyflo are you able to please provide an update for this?

**Comment by amyflo** [comment]
Hi @Olivia-Chiong ! I was on holiday and missed the meetings last week. 
Progress: looking through the Github / understanding documentation
Blockers: I am struggling a bit with understanding the documentation for the page
Availability: working on it!
ETA: unsure yet

**Comment by Olivia-Chiong** [comment]
### Overview
Create project file for true github contributors so that the project will appear when the website is launched

### Action Items
- [ ]
- [ ]

### Resources

**Comment by ExperimentsInHonesty** [comment]
@amyflo - here are the instructions on what you are supposed to do for this issue https://github.com/100Automations/Website/issues/31#issuecomment-752803377

**Comment by henlatourrette** [comment]
@mcspach will be working on this.

**Comment by mcspach** [comment]
UPDATE 
No progress because I've just been too busy with moving into my new apt. 
I will plan on getting some work in over the next few days.

**Comment by mcspach** [comment]
@NivenPrasad @Olivia-Chiong Adding a second card is causing some issues with the card section of our home page. 

I can:
1. try and solve these issues within the same pull request by adding new sass.
2. Push these changes to the new markdown file and then create a separate issue for the styling.

Which would you prefer?

**Comment by Olivia-Chiong** [comment]
Option 2 please, thank you!

**Comment by mcspach** [comment]
Incoming pull request will have:

- Added new .MD file for True Github Contributors which includes all the project information provided on this issue.
- Image was added to /assets folder and linked to in .MD

I will now open a new issue for fixing CSS for cards on Automations page.

**Comment by henlatourrette** [comment]
The pull request #205 will fix this issue.
